lib/internal/cluster/child.js

```diff
@@ -162,6 +164,21 @@ function rr(message, { indexesKey, index }, cb) {
 
   let key = message.key;
 
+  let fakeHandle = null;
+
+  function ref() {
+    if (!fakeHandle) {
+      fakeHandle = setInterval(noop, TIMEOUT_MAX);
```

Comment:
This seems very hacky. A comment here explaining what is going on here would be good for future generations who may find this because it's definitely not obvious.

Reply:
I have added a comment as follows.
```js
// Faux handle. net.Server is not associated with handle,
// so we control its state(ref or unref) by setInterval.
const handle = { close, listen, ref, unref };
```
do you mean we should  add a comment in `ref` function ?

Reply:
@jasnell  can you help review this again ? thanks.

Reply:
I'm going to merge this PR to main branch, i will submit another PR if necessary.